Assets/Scripts/Combat/Combantants/Combatant.cs

```diff
@@ -178,7 +179,7 @@ public void ApplyShield(int shieldHealth)
         if (MaxShieldPoints == 0)
         {
             Shield.SetActive(true);
-            Shield.transform.position = transform.position;
+            Shield.transform.position = transform.position + new Vector3(0.0f, ShieldPositionVerticalOffset, 0.0f);
```

Comment:
What I meant by my previous comment in the other commit was make the whole line ``new Vector3(0.0f, ShieldPositionVerticalOffset, 0.0f);`` a variable, not just the y position.

Reply:
No, a vector3 cannot be stored as a constant